src/components/ImageLoader.vue

```diff
@@ -116,13 +117,27 @@
    mounted: function() {
      this.parentDivWidth = this.$refs.parent.clientWidth;
      this.$refs.visibleImage.addEventListener('mousedown', this.onMouseDown);
-     this.$refs.visibleImage.addEventListener('wheel', this.onWheel);
      window.addEventListener('resize', this.onResize);
    },
    beforeDestroy: function() {
      window.removeEventListener('resize', this.onResize);
    },
    computed: {
+     routeAnnotationsCheck() {
```

Comment:
It would be better for separation of responsibility if this came as a prop from a parent component. Components become a lot harder to test and reuse if they actively *pull* information from their environment instead of passively *receiving* information from their props.

Also, as this is a getter, instead of this being named after what it checks, it would be better if it was named after what can be determined from this value. E.g. in this case `isHalfWidth` if it's a computed value (or `halfWidth` it were a prop)